src/app/_components/PageHeader.tsx

```diff
@@ -36,6 +36,18 @@ export function PageHeader({
   metaData,
   isFeatured = false,
 }: PageHeaderProps) {
+  function setDescriptionParagraph(description: string | string[]) {
```

Comment:
Can we define the `DescriptionProp` outside of the component with the rest of the type  definitions?

I also only noticed this but can we change the existing ones to `CTAProps` and `ImageProps` 🙏🏼